cyclr.js

```diff
@@ -204,6 +204,12 @@ function setRecord(record, datain) {
         var fieldValue = datain[fieldName];
 
         if (lineItems.indexOf(fieldName) > -1) {
+            // Update the subscriptions
+            if (fieldName === 'subscriptions') {

```

Comment:
Make `const subscriptionsFieldName = 'subscriptions';` global (top of the file) so it can be used here too?